src/components/map/MapService.js

```diff
@@ -1744,6 +1752,8 @@ goog.require('ga_urlutils_service');
                 layerTimestamps);
           }
 
+          gaTime.allowStatusUpdate = true;
```

Comment:
Yes exactly, I havn't tested but I think it could potentitally be the case.
Maybe we can trigger an event  gaLayersPermalinkLoaded.